README.md

```diff
@@ -1707,6 +1707,7 @@ API | Description | Auth | HTTPS | CORS |
 | [JSON2Video](https://json2video.com) | Create and edit videos programmatically: watermarks,resizing,slideshows,voice-over,text animations | `apiKey` | Yes | No |
 | [Lucifer Quotes](https://github.com/shadowoff09/lucifer-quotes) | Returns Lucifer quotes | No | Yes | Unknown |
 | [MCU Countdown](https://github.com/DiljotSG/MCU-Countdown) | A Countdown to the next MCU Film | No | Yes | Yes |
+| [Motivational Quotes](https://github.com/CrypticMessenger/QuoteAppAndAPI.git) | Random Motivational Quotes | No | Yes | Unknown |
```

Comment:
It's better to add [site's link](https://nodejs-quoteapp.herokuapp.com) rather than adding the link to the repo since the site is having documentation with quick demos.